src/csv.test.ts

```diff
@@ -140,6 +140,20 @@ test('should serialize from service when direction is from', (t) => {
   t.is(ret, expectedData)
 })
 
+test('should serialize array of data from service when flipped', (t) => {
+  const stateFlipped = { ...state, flip: true }
+  const data = [
+    { value: 1, text: 'Several words here', age: 39 },
+    { value: 2, text: 'And more here', age: 45 },
+    { value: 3, text: 'Even more', age: 81 },
+  ]
+  const expectedData = commaString
```

Comment:
I'm sure this is good but would love and explanation later.

Reply:
Sure. :)